scripts/tests/run_test_suite.py

```diff
@@ -126,6 +127,11 @@ class RunContext:
     multiple=True,
     help='What test tags to exclude when running. Exclude options takes precedence over include.',
 )
+@click.option(
+    '--path',
```

Comment:
path seems vague for a `run_test_suite.py` argument. If I see `run_test_suite.py --path foo` I would implicitly expect that path is what I should run.

How about naming this `--compiled-target-search-path` or similar?

Why is CHIP_ROOT a good default here? Should it not be `out` or something like that?